packages/odyssey-react/src/components/Table/Table.tsx

```diff
@@ -36,7 +37,7 @@ type ContainerProps =
       /**
        * The visible heading for the table
        */
-      title: ReactNode;
+      heading: ReactNode;
```

Comment:
the type annotation of `ReactNode` is a bit loose here, based on the way this is used within `TableContainer` we want something more strict like `ReactText`